src/black/__init__.py

```diff
@@ -2722,6 +2730,7 @@ class StringTransformer(ABC):
 
     line_length: int
     normalize_strings: bool
+    __name__ = "StringTransformer"
```

Comment:
What does this do?

Reply:
This gives a name to class-based transformers which otherwise would have to be queried with `__class__.__name__` instead.